### ISSUE TITLE
Reconstruction flipped when using `corrct` `astra` backend

### ISSUE THREAD

**Issue by harripj** [cicwi/PyCorrectedEmissionCT#12]
* PyCorrectedEmissionCT version: 0.4.2
* Python version: 3.9.7
* Operating System: Windows 10
* Astra verison: 2.1.0

### Description

The reconstructed volume is flipped along the first axis when using the `astra` backend.

### What I Did

Example:
```python
import corrct as cct
from skimage.data import shepp_logan_phantom
from skimage.transform import radon
from matplotlib import pyplot as plt
import numpy as np

angles = np.arange(180)
sino = radon(shepp_logan_phantom(), angles)
sino = sino.T

solver = cct.solvers.Sirt(verbose=True, regularizer=None)

vol_shape = [sino.shape[1], sino.shape[1]]
with cct.projectors.ProjectorUncorrected(vol_shape, np.deg2rad(angles), use_astra=True) as A:
    rec_cct_astra, _ = solver(A, sino, 10, lower_limit=None)

with cct.projectors.ProjectorUncorrected(vol_shape, np.deg2rad(angles), use_astra=False) as A:
    rec_cct_skimage, _ = solver(A, sino, 10, lower_limit=None)

fig, ax = plt.subplots(ncols=3, figsize=(12, 4))
ax[0].matshow(shepp_logan_phantom())
ax[1].matshow(rec_cct_astra)
ax[2].matshow(rec_cct_skimage)

ax[0].set_title('Original')
ax[1].set_title('CCT astra')
ax[2].set_title('CCT skimage')
```
![image](https://user-images.githubusercontent.com/16853829/160854484-e9932ed8-a0bc-434b-8fb8-c4d49d7fb938.png)

The image orientation is correct when just using the `astra` toolbox:
```python
import astra

det_width = 1
# initialize geometries
proj_geom = astra.create_proj_geom(
    "parallel", det_width, sino.shape[-1], np.deg2rad(angles)
)

vol_shape = (sino.shape[-1], sino.shape[-1])
# create reconstructed volume geometry
vol_geom = astra.creators.create_vol_geom(*vol_shape)
# create projector
projector_id = astra.create_projector("linear", proj_geom, vol_geom)

# create sinogram from current slice projection data
projection_id = astra.data2d.create("-sino", proj_geom, sino)
# create volume for reconstructed data, initialised to 0
reconstruction_id = astra.data2d.create("-vol", vol_geom, 0)

cfg = astra.astra_dict("SIRT")
cfg["ProjectorId"] = projector_id
cfg["ProjectionDataId"] = projection_id
cfg["ReconstructionDataId"] = reconstruction_id
# cfg.update(kwargs)
# create the algorithm and run
algorithm_id = astra.algorithm.create(cfg)
astra.algorithm.run(algorithm_id, 10)
# add slice to output
rec_astra = astra.data2d.get(reconstruction_id)
# cleanup
astra.algorithm.delete(algorithm_id)
astra.data2d.delete(reconstruction_id)
astra.data2d.delete(projection_id)
```
![image](https://user-images.githubusercontent.com/16853829/160857809-b53b2985-19d7-4eac-9e8b-bbf2371661f2.png)

I have tried the `Sirt` and `PDHG` solvers in `corrct` and both reconstructions are flipped.

**Comment by Obi-Wan** [comment]
Thank you for reporting this. It should now be fixed on `master` through 96d1b7e0344d5a9f075d0358457a05d41dfd8c83.

Probably, it's time to prepare a new point release... :)

**Comment by harripj** [comment]
Excellent, thanks for the quick fix!
